website/app/components/doc/page/footer.hbs

```diff
@@ -1,3 +1,15 @@
 <footer class="doc-page-footer" ...attributes>
-  <LinkTo @route="testing">Testing</LinkTo>
+  <div class="doc-page-footer__links doc-text-body-small">
+    <a href="#">Accessibility</a>
+    <a href="#">Careers</a>
+    <a href="#">Cookie Manager</a>
+    <a href="#">Terms of Use</a>
+    <a href="#">Privacy</a>
+    <span role="presentation">–</span>
+    <LinkTo @route="testing">Testing</LinkTo>
+    <button type="button" class="doc-page-footer__🌈" {{on "click" this.toggleLayoutColors}}>🌈</button>
+  </div>
+  <a class="doc-page-footer__logo" href="https://www.hashicorp.com/" target="_blank" rel="noopener noreferrer">
```

Comment:
```suggestion
  <a class="doc-page-footer__logo" href="https://www.hashicorp.com/" target="_blank" rel="noopener noreferrer" aria-label="hashicorp dot com">
```

Reply:
fixed in https://github.com/hashicorp/design-system/pull/807/commits/29273a669cc075b4d3e768c3c1086bc58119007a